test/EdgeChangerTest.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file   EdgeChangerTest.cpp
+ * @brief  EdgeChangerクラスのテスト
+ * @author mutotaka0426
+ */
+
+#include "EdgeChanger.h"
+#include <gtest/gtest.h>
+#include <cmath>
+
+namespace etrobocon2022_test {
+
+  TEST(EdgeChanger, runFalseToFalse)
+  {
+    bool isLeftEdge = false;  // run()を実行することでこの変数が切り替わる
```

Comment:
参照渡ししていることを書いたほうがより理解度が上がるかも?

```suggestion
    bool isLeftEdge = false;  // run()を実行することでこの変数が切り替わる(コンストラクタで参照渡しをするため)
```